### PR TITLE
Update README.md (SASS deprecation)

### PR DESCRIPTION
**Pull request by ffxsam**
Updated SASS code to not used deprecated methods.

### Description
Please explain the changes you made here. Include an example of what your changes fix or how to use the changes.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] All tests passing (`yarn test`)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,18 +247,20 @@ and so on.
 Better yet, here is a SASS code snippet which can make life much easier (Thanks to @ascendantofrain, [#81](https://github.com/gridstack/gridstack.js/issues/81) and @StefanM98, [#868](https://github.com/gridstack/gridstack.js/issues/868)) and you can use sites like [sassmeister.com](https://www.sassmeister.com/) to generate the CSS for you instead:
 
 ```sass
-@use "sass:math";
 .grid-stack > .grid-stack-item {
+  $gridstack-columns: 2;
```
Comment (adumesny) [inline README.md:251]: hey thanks, can you put it back to 12 and make the width & left 1 line again ?

**Comment by adumesny** [comment]
really it's deprecated now ? that was updated not so long ago to follow https://sass-lang.com/documentation/breaking-changes/slash-div

also you are missing the gridstack.scss and other files if really deprecated.... but it works just fine

**Comment by ffxsam** [comment]
Yeah, I got warnings about using those calculations. You could also use `math.div`, but I found `calc()` to look cleaner.

I'll make those changes in a sec.

**Comment by ffxsam** [comment]
Done!

**Comment by leemoria** [comment]
> really it's deprecated now ? that was updated not so long ago to follow https://sass-lang.com/documentation/breaking-changes/slash-div
> 
> ```css-scss
> // RIGHT, will work in future Sass versions.
> @debug math.div(12px, 4px); // 3
> ```
> 
> also you are missing the gridstack.scss and other files if really deprecated.... but it works just fine

When the latest version is published? thanks. My angular Project is Wrong

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# gridstack.js
+be# gridstack.js
```
Comment (adumesny) [inline README.md:1]: looks like junk char. will remove...

**Pull request merged**